tests/test_embedding_chain_strengths.py

```diff
@@ -18,7 +18,7 @@
 
 import dimod
 import dwave.embedding
-from dwave.embedding.chain_strengths import uniform_torque_compensation
+from dwave.embedding.chain_strengths import uniform_torque_compensation, scaled_to_problem
```

Comment:
Maybe we should use singular for this module/namespace? @arcondello?

Reply:
Agree. And this hasn't made it into a release yet so no backwards compatibility break.